packages/terra-clinical-alert/tests/nightwatch/AlertTests.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Link } from 'react-router';
+
+const AlertTests = () => (
+  <div>
+    <ul>
+      <li><Link to="/tests/alert-tests/default">Alert Default</Link></li>
+      <li><Link to="/tests/alert-tests/type">Alert Type</Link></li>
+      <li><Link to="/tests/alert-tests/title">Alert Title</Link></li>
```

Comment:
Looking at the example's implementation, I see type and title both mean different thing, but I wasn't sure what these links intended to show